pdxJaxon/UMM2
# Client/build team selector

#### Changes

- Splits the `start` command into 3 command `start:api`, `start:client` and `start` (which runs both at once)

- Implements a basic "store" allowing data to be stored locally

- Implements a reducer, actions, and a selector, segregating the data layer from the ui 

- Implements a service section to fetch data from the api

- Implements a basic team selector

#### Testing

- Start by making sure the teams table is _empty_. Start the api with `npm run start:api` and the client with `npm run start:client`.

- Once the UI loads, expect to see a message saying no teams were found

- Insert a team into the DB with:
```
insert into team (
  teamId, abbr, city, nickname, createdAt, updatedAt
) values (
  1, 'abc', 'Portland', 'abc', date(), date()
);
```

- Expect to see the option in the drop down

- Change the team name, and insert another

- Expect to see that in the dropdown.

- Select an option, and expect that team name to be displayed on the screen
